EasyPost.Tests/HttpTests/RequestTest.cs

```diff
@@ -17,5 +18,37 @@ public void TestRequestDisposal()
         CustomAssertions.DoesNotThrow(() => request.Dispose());
     }
 
+    [Fact]
+    public void TestQueryParameterList()
+    {
+        const string key = "key";
+        var value = new List<string>
+        {
+            "value1",
+            "value2"
+        };
+        const string key2 = "key2";
+        const string value2 = "value3";
+        var parameters = new Dictionary<string, object>
+        {
+            { key, value },
+            { key2, value2 }
+        };
+
+        Request request = new("https://google.com", "not_a_real_endpoint", Method.Get, ApiVersion.V2, parameters, new Dictionary<string, string> { { "header_key", "header_value" } });
+
+        HttpRequestMessage httpRequestMessage = request.AsHttpRequestMessage();
+
+        string queryString = httpRequestMessage.RequestUri?.Query;
+
+        string keyWithArray = $"{key}[]";
+
+        foreach (string item in value)
+        {
+            Assert.Contains($"{keyWithArray}={item}", queryString); // Does not account for URL encoding
```

Comment:
issue: assertions should be explicit and hard-coded. Otherwise this test will always pass. The assertion is the blueprint, the thing we actually expect which shouldn't be dynamic for that reason.

Reply:
> Otherwise this test will always pass

The test is not asserting the output, the test is asserting a given input produces an output in a specific format, where the input is irrelevant.

Regardless, resolved.